Bottom: d3cfd0a08a93367d126e26d3e5e4354d7960c189
Top:    f6ccfaa1deb4fbb018c5dbc6a9b25a9285c0c156
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-14 19:59:55 +0000

Refresh of gui-netlist-windows-set-find-f

---

diff --git a/src/hid/lesstif/netlist.c b/src/hid/lesstif/netlist.c
index c53a9b7..6687341 100644
--- a/src/hid/lesstif/netlist.c
+++ b/src/hid/lesstif/netlist.c
@@ -132,7 +132,7 @@ nbcb_select_common (LibraryMenuType *net, int pos, int select_flag)
   ConnectionType conn;
   int i;
 
-  SaveFindFlag (FOUND);
+  SaveFindFlag (FOUNDFLAG);
   InitConnectionLookup ();
   ResetConnections (true);
